tools/build/Makefile.vars

```diff
@@ -82,10 +82,15 @@ LIB_ASAN_PATH =
 PREFIX := $(patsubst "%",%,$(PREFIX))
 EXTRA_BINS :=
 
+SOL_VERSION := 0
+SOL_CURRENT := 0
+SOL_REVISION := 0
+SOL_AGE := 0
+
 export PKGNAME := soletta
-export VERSION_MAJOR := 0
-export VERSION_MINOR := 0
-export VERSION_RELEASE := 1
+export VERSION_MAJOR := $(shell echo $$(($(SOL_CURRENT) - $(SOL_AGE))))
```

Comment:
We're not using gmsl on Soletta.
So we would need to include it only to make a subtraction.

On the other hands we have about a dozen of shell calls in our makefile.

I believe we're fine with current approach